playground/playground.js

```diff
@@ -44,9 +44,6 @@
 
   // map of original to modifed contexts
   playground.contextMap = {
-    // be careful when working with redirectors as as Chrome (not firefox)
-    // will drop Accept: application/ld+json after redirecting
```

Comment:
Should leave the comments.  They (maybe) apply to future rules that might be added.